src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -308,6 +308,18 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
             return;
         }
         const replyingToThread = this.props.relation?.key === THREAD_RELATION_TYPE.name;
+        const navAction = getKeyBindingsManager().getNavigationAction(event);
+        switch (navAction) {
+            case KeyBindingAction.SelectPrevMessage:
+                (
+                    document.querySelector<HTMLElement>(".mx_EventTile_selected") ||
+                    document.querySelector<HTMLElement>(".mx_EventTile_last")
+                )?.focus();
```

Comment:
Ideally this would be not done via query selectors and instead perform a dispatch which the correct Element reacts to and focuses itself, if that is infeasible could we have a comment explaining why

Reply:
Done in https://github.com/matrix-org/matrix-react-sdk/pull/12328/commits/34135cb5f4a4946da4085536dea1440f4646313e